torch/ao/ns/fx/mappings.py

```diff
@@ -450,7 +450,7 @@ def get_node_type_to_io_type_map() -> Dict[str, Set[NSNodeTargetType]]:
         F.mish,
         # TODO(future PR): implement shadowing for binary ops and
         # uncomment below
-        # operator.add,
+        operator.add,
```

Comment:
Hi @vkuzo is this OK? or should I skip the check in numeric suite test

Reply:
sg

Reply:
Is the above TODO out of date then?

Reply:
there is a mul below still not enabled yet, I can move the comment